modules/services/imapnotify.nix

```diff
@@ -34,6 +34,28 @@ let
       };
     };
 
+  genAccountAgent = account:
+    let name = safeName account.name;
```

Comment:
`safeName` converts `hw@example.com` to `hw-example.com` but for the `plist` label it should arguably replace the `.` with a `-` too (for `hw-example-com`).